tests/foreman/api/test_syncplan.py

```diff
@@ -607,8 +607,12 @@ def validate_repo_content(
             except AssertionError:
                 sleep(30)
         else:
-            raise AssertionError(
-                'Repository contains invalid number of content entities')
+            repo = repo.read()
+            self.assertNotEqual(
```

Comment:
sure, this comes from experimentation with information api provides for repo. If the repo was never synced and is not beeing synced `last_sync` equals to None. If the repo is being synced at the moment of query `last_sync` equals to Pending. So this assert aims to not fail when sync is in progress.